### PR TITLE
Used EG(active) instead of Removed EG(valid_symbol_table) - 7.2.0alpha3

### PR DESCRIPTION
**Pull request by remicollet**
Related to https://bugs.xdebug.org/view.php?id=1452
See http://git.php.net/?p=php-src.git;a=commitdiff;h=caaeb4849aa56cbbdc66ea015c11a58bd47a43ff

### CODE REVIEW

**Comment by smalyshev** [comment]
Should this be added to UPGRADING.INTERNALS?

**Comment by remicollet** [comment]
@smalyshev indeed, done

**Comment by Jan-E** [comment]
For reference: https://github.com/php/php-src/commit/8d1ae4e1c5bc3226df0436abf838652c1694fe95

File: xdebug_stack.c [hunk 1]
```diff
@@ -800,7 +800,11 @@ void xdebug_error_cb(int type, const char *error_filename, const uint error_line
 			break;
 	}
 
+#if PHP_VERSION_ID >= 70200
+	if (PG(track_errors) && EG(active)) {
```
Comment (sgolemon) [inline xdebug_stack.c]: EG(active) has been available for quite awhile.

You can probably get away with *just* using this version and not bothering with the valid_symbol_table check at all.

Comment (remicollet) [inline xdebug_stack.c]: I'm too careful ;)   

(who say paranoid ?)

**Comment by derickr** [comment]
Merged - thanks for the fix!

**Pull request merged**